SourceCode/GPS/Forms/Sections.Designer.cs

```diff
@@ -18,26 +18,35 @@ public partial class FormGPS
         public btnStates manualBtnState = btnStates.Off;
         public btnStates autoBtnState = btnStates.Off;
 
-        private void markAsWorkedTrack()
+        private void MarkAsWorkedTrack()
         {
-            //is it an AB Line or a Curve?
-            if (trk.gArr[this.trk.idx].mode == TrackMode.AB)
+            // only mark if there was a track selected
+            if (this.trk.idx >= 0)
```

Comment:
Maybe use an early return here:
```suggestion
            if (this.trk.idx < 0) return;
```

Reply:
And also remove the `{` afterwards. (That's the whole idea of the early return: to get rid of an indentation level 😉)